### ISSUE TITLE
fix: ARIA label for panel/dialog dismiss button tooltip

### ISSUE THREAD

**Issue by philliphoff** [microsoft/fluentui-blazor#3816]
# 🐛 Bug Report

The dismiss button tooltip for panels/dialogs does not have an ARIA label (`aria-label`) attribute.

## 💻 Repro or Code Sample

This can be repro'd using the Fluent UI demo site: https://www.fluentui-blazor.net/Panel

1. Open one of the panels (with a dismiss button enabled)
2. Examine the tooltip element in the browser developer tools

## 🤔 Expected Behavior

The tooltip element should have an `aria-label` (set to the same value as the tooltip?)

## 😯 Current Behavior

The tooltip element does not have an `aria-label`.

## 💁 Possible Solution

Instead of using a FluentUI tooltip, perhaps add the `DismissTitle` as the `Title` of the dismiss button, where the browser will display a tooltip using its native UX. (Admittedly, not as pretty as the FluentUI tooltip UX.)

## 🔦 Context

This was flagged in an accessibility review.

## 🌍 Your Environment

.NET 9, WASM, Fluent UI library v4.11.9.

* OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
* Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
* .NET and Fluent UI Blazor library Version [e.g. 8.0.2 and 4.4.1]

**Comment by vnbaaij** [comment]
There is an aria-label. But it is attached to the button that the tooltip attaches to:

![Image](https://github.com/user-attachments/assets/5ac5e2d7-8517-4964-b135-716b7ec66771)

And the generated source:

![Image](https://github.com/user-attachments/assets/3dd8f346-479d-4e14-89b3-73769c2191de)

This has not been flagged as an issue in Aspire and other a11y reviews before.

**Comment by philliphoff** [comment]
@vnbaaij The Accessibility Insights for Web (Fast Pass) tool does flag this issue, both on the FluentUI demo site as well as within the Aspire dashboard.  I can't speak to why it wasn't brought up in other reviews, only that it was in ours.  While I do see the `aria-label` on the button, perhaps the analysis tool would like to see the `aria-labeledby` on the tooltip as well that references the button?

<img width="1796" alt="Image" src="https://github.com/user-attachments/assets/a4ae761e-457b-45ad-8fee-ad4747c451a0" />
